change/@microsoft-teams-js-7e2d7681-be7b-4c3c-b196-470aa65092c4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Currently if a child IFrame is opened and the TeamsJs SDK is used it will not receive all the events that are sent to the parent IFrame.  This change allows the proxying of events if they are not handled in the parent frame.",
```

Comment:
>Currently if a child IFrame is opened and the TeamsJs SDK is used it will not receive all the events that are sent to the parent IFrame.  This change allows the proxying of events if they are not handled in the parent frame. [](http://example.com/codeflow?start=14&length=224)

For consistency among changelog messages, can you rephrase to something like this (feel free to copy if you want):

"Enabled proxying of window events to child frames if unhandled by current frame."